modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_storage.es.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import formatStorage from '../../../src/main/resources/META-INF/resources/liferay/util/format_storage.es';
+
+describe('Liferay.Util.formatStorage', () => {
+	it('should return formatted string with size and suffix type KB if size input type is number <= 1048575 and options is default value', () => {
```

Comment:
When describing tests, reduce description only the main purpose of the test, there is no need to list all states.

`'should format size under 1048575 bytes to kilobytes, with the default 'KB' suffix'`